gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -95,6 +96,25 @@
    */
   public static final String COPY_TARGET_TABLE_ROOT =
       HiveDatasetFinder.HIVE_DATASET_PREFIX + ".copy.target.table.root";
+  
+  /**
+   * These two options, in pair, specify the output location of the data files on copy
+   * {@link #COPY_TARGET_TABLE_PREFIX_TOBE_REPLACED} specified the prefix of the path (without Scheme and Authority ) to be replaced
+   * {@link #COPY_TARGET_TABLE_PREFIX_REPLACEMENT} specified the replacement of {@link #COPY_TARGET_TABLE_PREFIX_TOBE_REPLACED}
+   * <p>
+   * for example, if the data file is hdfs://ltx1-unonn01.grid.linkedin.com:9000/data/databases/ANET/ANET_CONTACTS_MANIFEST/1462826996005-PT-2085190/part-00000.avro , 
```

Comment:
Please remove this from the comment (directory structure is internal information), and squash the commits to make it disappear from git history.